app/controllers/admin/organizations_controller.rb

```diff
@@ -25,8 +25,7 @@ def new
       @organization.user = User.new
     end
 
-    def edit
-    end
+    def edit() end
```

Comment:
I think the `def edit; end` syntax is better here, was this a rubocop autocorrection or manual fix?

Reply:
I fixed manually to fix a rubocop offense but I'll change in the next commit.

Reply:
You can just rebase the commit to fix it 👍